sr_x86_64/libydberrors.h

```diff
@@ -0,0 +1,1502 @@
+/****************************************************************
```

Comment:
The following dangling line at the very end of the commit message can be removed.

    - sr_x86_64/libydberrors.h

Reply:
This was a new file that was missing its description (have to check I didn't miss others after I do the big squish)